server/statshouse/statshouse-client.cpp

```diff
@@ -47,6 +47,18 @@ class statshouse_stats_t : public stats_t {
     add_stat(type, key, static_cast<double>(value));
   }
 
+  void add_stat_with_tag_type(char type [[maybe_unused]], const char *key, const char *type_tag, double value) noexcept final {
+    std::vector<std::pair<std::string, std::string>> metric_tags = {{"type", std::string(type_tag)}, {"host", std::string(kdb_gethostname())}};
+    auto metric = make_statshouse_value_metric(normalize_key(key, "_%s", stats_prefix), value, {});
```

Comment:
It seems, you forgot to pass tags :)
```suggestion
    auto metric = make_statshouse_value_metric(normalize_key(key, "_%s", stats_prefix), value, metric_tags);
```